File: app/src/main/java/panteao/make/ready/beanModel/responseModels/LoginResponse/LoginResponseModel.java
package panteao.make.ready.beanModel.responseModels.LoginResponse;

public class LoginResponseModel {

    //@SerializedName("data")
    private UserData data;

    //@SerializedName("responseCode")
    private int responseCode;
    private boolean status;
    private String debugMessage;

    public boolean isStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public Object getDebugMessage() {
        return debugMessage;
    }

    public void setDebugMessage(String debugMessage) {
        this.debugMessage = debugMessage;
    }

    public UserData getData() {
        return data;
    }

    public void setData(UserData data) {
        this.data = data;
    }

    public int getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(int responseCode) {
        this.responseCode = responseCode;
    }

    @Override
    public String toString() {
        return
                "LoginResponseModel{" +
                        "data = '" + data + '\'' +
                        ",responseCode = '" + responseCode + '\'' +
                        "}";
    }
}

File: enveusdk/src/main/java/com/make/userManagement/bean/LoginResponse/LoginResponseModel.kt
package com.make.userManagement.bean.LoginResponse

class LoginResponseModel {

    //@SerializedName("data")
    var data: Data? = null

    //@SerializedName("responseCode")
    var responseCode: Int = 0
    var isStatus: Boolean = false
    private var debugMessage: String? = null

    fun getDebugMessage(): String? {
        return debugMessage
    }

    fun setDebugMessage(debugMessage: String) {
        this.debugMessage = debugMessage
    }

    override fun toString(): String {
        return "LoginResponseModel{" +
                "data = '" + data + '\''.toString() +
                ",responseCode = '" + responseCode + '\''.toString() +
                "}"
    }
}